pages/search_hotels_page.py

```diff
@@ -0,0 +1,41 @@
+import allure
+from pages.base_page import BasePage
+from utils.browser_helper import BrowserHelper
+from .locators.locators import SearchHotelsPageLocators
+
+
+class SearchHotelsPage(BrowserHelper, BasePage):
+    @allure.step("Accept cookies")
+    def accept_cookies(self):
```

Comment:
it should be one method for all pages

Reply:
We have different types of pop-ups with different locators on different pages. For implementation with single usage for all pages, I see one way - use method by BDD approach.  I mean we will have to initialize the page every time when we use the method "accept_cookies".  It seems too complex.